DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -194,5 +194,40 @@ public void UpdateLearningPathwayDefaultsForCourse_calls_data_service()
             A.CallTo(() => courseDataService.UpdateLearningPathwayDefaultsForCourse(1, 6, 12, true, true))
                 .MustHaveHappened();
         }
+
+        [Test]
+        public void DoesCourseNameExistAtCentre_calls_data_service()
+        {
+            // When
+            courseService.DoesCourseNameExistAtCentre("Name", 101, 1);
+
+            // Then
+            A.CallTo(() => courseDataService.DoesCourseNameExistAtCentre("Name", 101, 1))
+                .MustHaveHappened();
+        }
+
+        [Test]
+        public void UpdateCourseDetails_calls_data_service()
+        {
+            // Given
+            A.CallTo(

```

Comment:
Small change. We can be more specific here and replace A<int>._ with the values to test that the data service method is called with the data that is passed in, rather than any ID which would pass A._.

Reply:
Sure, made the change, and solved merge conflicts :)

Reply:
Thank you